src/site/content/en/lighthouse-accessibility/list/index.md

```diff
@@ -1,43 +1,41 @@
 ---
 layout: post
-title: Ensure lists are structured correctly
+title: "Lists do not contain only `<li>` elements and script supporting elements (`<script>` and `<template>`)"
 description: |
-  Learn about list audit.
+  Learn how to make lists on your web page clear to screen reader users by
+  avoiding child elements that don't belong in lists.
 date: 2019-05-02
+updated: 2019-09-19
 web_lighthouse:
   - list
 ---
 
-Screen readers depend upon lists being structured properly,
+Screen readers depend on lists being structured properly
 to keep users informed of content within the lists.
 The only content lists should contain should be within `<li>` elements.
 They can also contain script supporting elements (`<script>` and `<template>`).
-Lighthouse reports when lists contain any other content elements that shouldn't be within the lists:
+
+## How this Lighthouse audit fails
+
+Lighthouse flags lists that contain content elements that shouldn't be in a list:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="list.png" alt="Lighthouse audit showing lists contain content elements that shouldn't be within the lists
-  <figcaption class="w-figcaption">
-    Lists contain content elements that shouldn't be within the lists.
-  </figcaption>
+  <img class="w-screenshot" src="list.png" alt="Lighthouse audit showing lists contain content elements that shouldn't be within the lists">
 </figure>
 
-## How to fix this problem
+{% include 'content/lighthouse-accessibility/scoring.njk' %}
 
-To fix this problem,
-remove any elements in lists that don't belong there.
+## How to create correctly structured lists
+
+Remove any elements in lists that don't belong there.
 Ordered and unordered lists must only contain `<li>`, `<script>` or `<template>` elements.
 
-Valid lists must have parent elements (`ul` or `ol` elements) and child elements (`li` elements).
+Valid lists must have parent elements (`<ul>` or `<ol>` elements) and child elements (`<li>` elements).
 Any other content elements are invalid.
 Learn more in
```

Comment:
Just noticed this, but I think many audits in this section contain a link to the deque doc, and then another link to it in the Resources section immediately after.

Reply:
Yup. As noted on another comment, that's something I was cleaning up as I did a quality pass on each section. I can go ahead and remove those globally.